data-migrations/2024-09-25-1120-create-questions.sql

```diff
@@ -62,7 +62,7 @@ CREATE TABLE `versionedQuestions` (
   INDEX versionedQuestions_section_idx (`versionedSectionId`, `displayOrder`)
 ) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb3;
 
-CREATE TABLE `versionedQuestionCondition` (
+CREATE TABLE `versionedQuestionConditions` (
```

Comment:
@briri: I believe that "condition" needs to still be updated to conditionType on line 70.